mongo_connector/connector.py

```diff
@@ -781,12 +782,20 @@ def apply_namespaces(option, cli_values):
                 "Destination namespaces set should not"
                 " contain any duplicates.")
 
-        for key, value in dest_mapping.items():
-            if key.count("*") > 1 or value.count("*") > 1:
+        for source_name, value in dest_mapping.items():
+            # Mapping may be old style with only a target name, eg:
+            # "db.source": "db.dest",
+            # or new style which can support extra options per namespace:
+            # "db.source": {"rename": "db.dest", ...}
+            if isinstance(value, dict):
+                dest_name = value.get("rename", source_name)
```

Comment:
I like the second example. For simply including a collection, it may not matter what the value is (e.g. could be ``True``).

If we didn't mind changing the config file format a bit more, we could get rid of namespaces.(include|exclude) entirely like this:

```json
{
  "namespaces": {
    "included.collection": true,
    "excluded.collection": false,
    "renamed.collection": "something.else"
  }
}
```